docs/repo-docs/reference/configuration.mdx

```diff
@@ -117,6 +117,47 @@ To help with incremental migration or in situations where you can't use the `pac
   environment variable.
 </Callout>
 
+### `cacheDir`
+
+Default: `".turbo/cache"`
+
+Specify the filesystem cache directory.
+
+```jsonc title="./turbo.json"
+{
+  "cacheDir": ".turbo/cache"
+}
+```
+
+### `daemon`
+
+Default: `false`
```

Comment:
```suggestion
Default: `true`
```

Reply:
When I try this locally it appears to be false by default:

![Screenshot_20240817_145952](https://github.com/user-attachments/assets/f3841d0a-3a9a-4f22-b01f-6782ffce6e49)

Also the config command returns null ([we have a test for this](https://github.com/vercel/turborepo/blob/219c1140d781a7bd5657e7d718e5e96247c8b00c/turborepo-tests/integration/tests/config.t#L66)) by default (not true).

Also my reading of [the code where this value is finally consumed](https://github.com/vercel/turborepo/blob/219c1140d781a7bd5657e7d718e5e96247c8b00c/crates/turborepo-lib/src/run/builder.rs#L263) is that if it's not `Some(true)`, then the daemon won't be used.

Am I missing something?

Reply:
Oh, wow, interesting. I think this is meant to be on by default. You're right, but we may want to follow up on this as a team.

Reply:
update: after internal investigation we determined that it is on by default and the above screenshot where I determined that it isn't was due to some other problem or error.  Also, I was _very close_ on the code I believed to be responsible for turning the daemon on, but that was the code for _package discovery_ and the _actual_ place was [a few lines above that](https://github.com/vercel/turborepo/blob/219c1140d781a7bd5657e7d718e5e96247c8b00c/crates/turborepo-lib/src/run/builder.rs#L207).

this was fixed in dcc630e7f07828e25a53697073387895752a297b